Airports/EGNR/SMR/Labels.txt

```diff
@@ -1,7 +1,11 @@
 ;Holding Points
-"A"	N053.10.43.235 W002.58.51.238 standHold
-"B"	N053.10.33.833 W002.58.39.367 standHold
-"C"	N053.11.04.374 W002.57.59.446 standHold
-"D"	N053.10.37.456 W002.58.56.321 standHold
-"G"	N053.10.50.360 W002.58.33.860 standHold
-"N"	N053.11.07.526 W002.58.09.446 standHold
\ No newline at end of file
+"E1" N053.10.42.806 W002.58.29.308 standHold
+"B1" N053.10.35.561 W002.58.40.133 standHold
+"A1" N053.10.42.653 W002.58.50.693 standHold
+"A2" N053.10.44.718,W002.58.54.422 standHold
+"D2" N053.10.44.136,W002.58.59.225 standHold
+"D1" N053.10.37.493,W002.58.56.574 standHold
+"G1" N053.10.49.904,W002.58.34.473 standHold
+"N1" N053.11.05.897,W002.58.09.536 standHold
+"C1" N053.11.04.084,W002.57.59.873 standHold
+"C2" N053.11.03.344,W002.57.59.032 standHold
```

Comment:
Please put these in alphabetical order, and remove the commas from the coordinates

Reply:
done